### PR TITLE
Expand chart

### PR DESCRIPTION
**Pull request by pflopez**
Expand action on stands chart, plus:
- Fixed some css issues with encapsulation none
- Moved some components around so we have a flat tree on `treatments`
- Removed providedIn: 'root', from DirectImpactsStateService, which was wrong
- Refactored stands chart to encapulate chart and empty state on one component

### CODE REVIEW

File: src/interface/src/app/treatments/expanded-stand-data-chart/expanded-stand-data-chart.component.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import "mixins";
+
+.expanded-stand-data {
+  padding: 20px;
+  height: 70vh;
+}
+
+::ng-deep {
+  sg-modal.stand-data-modal {
```
Comment (pflopez) [inline src/interface/src/app/treatments/expanded-stand-data-chart/expanded-stand-data-chart.component.scss:9]: @lastminutediorama I couldn't find this exact variant on the modal component. We have `shortHeader` but this one is a bit bigger according to figma, and font styles a tad different.  Not sure if we should just add one more thing or audit the headers and see if we need to refactor.
Also the close icon is different in figma for these "expanded" panels... maybe its just another, very similar, component?

Comment (lastminutediorama) [reply src/interface/src/app/treatments/expanded-stand-data-chart/expanded-stand-data-chart.component.scss:9]: I'm not sure either. It's probably worth considering a refactor of the headers.

Comment (lastminutediorama) [reply src/interface/src/app/treatments/expanded-stand-data-chart/expanded-stand-data-chart.component.scss:9]: But to be clear, I'm okay with whatever you want to do for this PR.

**Review by lastminutediorama — APPROVED** [review]
Looks good--the expansion approach looks really clean.
I have one concern though, and I'm not sure if it's important, but the Figma shows this expanded dialog at ~1000px width, which might mean this component is meant to be a "zoomed-in" view of the chart. I don't know if that's the intention, and I don't know if that means we'll need to adjust the way we render some of the charts, but I thought I should mention it.

**Review by LALuis — APPROVED** [review]

**Comment by pflopez** [comment]
> Looks good--the expansion approach looks really clean. I have one concern though, and I'm not sure if it's important, but the Figma shows this expanded dialog at ~1000px width, which might mean this component is meant to be a "zoomed-in" view of the chart. I don't know if that's the intention, and I don't know if that means we'll need to adjust the way we render some of the charts, but I thought I should mention it.

Good catch, I used  `width="large"` but that might be wrong, Though Im pretty sure we'll need to revisit this based on the other changes on the modal I mentioned

**Pull request merged**